src/styles/preset.css

```diff
@@ -1,43 +1,47 @@
 @import "tailwindcss" prefix(oui);
 @import "tw-animate-css";
 
+/* Google Fonts */
+@import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,200..900;1,200..900&display=swap');
```

Comment:
we should not import from an external source. we need to include the fonts locally